### DISCUSSION TITLE
is it possible to merge two react app into one nx mono repository?

### DISCUSSION THREAD

**Discussion by melonges** [nrwl/nx#17718] [Feature Requests]
is it possible to merge two react-vite projects into one nx mono repository?

**Comment by lazee** [comment]
Depends on what you mean by "merge". But yes, you can easily have as many react applications as you like inside Nx. That is kind of the whole point :)

You can choose two strategies for your repo, package based and integrated repo. Read more about it here: https://nx.dev/getting-started/intro

**Comment by AgentEnder** [comment]
Yeah, see: https://nx.dev/recipes/adopting-nx/preserving-git-histories#merging-in-a-standalone-project
